vendor/bat-native-ads/src/bat/ads/internal/account/account.cc

```diff
@@ -174,6 +174,10 @@ void Account::OnDidReconcileAdRewards() {
 void Account::OnDidRedeemUnblindedPaymentTokens() {
   BLOG(1, "Successfully redeemed unblinded payment tokens");
 
+  if (ConfirmationsState::Get()->get_unblinded_payment_tokens()->IsEmpty()) {
```

Comment:
@tmancey why early return here?

Reply:
@moritzhaller We are early returning because we have no unblinded payment tokens to remove on line 184. And as there are no unblinded payment tokens there will be no uncleared transactions.